docs/content/Modpacks/Examples/Superheated_Pyrolyzing_Oven.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Superheated Pyrolyzing Oven"
+---
+
+### Superheated Pyrolyzing Oven Multiblock (by Phoenixvine)
+
+In order to use multiblock logic extending beyond the normal WorkableElectricMultiblockMachine, (This is the multiblock type used by default for kubejs) you need to load a class. Coil multiblocks such as the Electric Blast Furnace, Pyrolyse Oven, and the Cracker use this class.
+
+```const CoilWorkableElectricMultiblockMachine = Java.loadClass("com.gregtechceu.gtceu.api.machine.multiblock.CoilWorkableElectricMultiblockMachine")```
+
+Below is an example of a multiblock using the CoilWorkableElectricMultiblockMachine class and the pyrolyseOvenOverclock machine logic.
+
+### Multiblock
+```js title="superheated_pyrolyzing_oven_multiblock.js"
+GTCEuStartupEvents.registry('gtceu:machine', event => {
+    event.create("superheated_pyrolyzing_oven", "multiblock")
+        .rotationState(RotationState.NON_Y_AXIS)
+        .recipeTypes('pyrolyse_oven')
+        .recipeModifiers([GTRecipeModifiers.PARALLEL_HATCH, GTRecipeModifiers.OC_PERFECT,GTRecipeModifiers.BATCH_MODE, (machine, recipe) =>
+        .machine((holder) => new CoilWorkableElectricMultiblockMachine(holder))
+        GTRecipeModifiers.pyrolyseOvenOverclock(machine, recipe)])
```

Comment:
Something went wrong here. I think the .machine is in the wrong spot. Take that out and put it below the event.create